### PR TITLE
Metaweather alternative

### PR DESCRIPTION
**Pull request by nick-gravgaard**
[metaweather.com](http://metaweather.com) is down. This PR changes the code to use [open-meteo.com](http://open-meteo.com)'s API and display [open source weather icons](https://github.com/erikflowers/weather-icons) instead.

### CODE REVIEW

File: src/Server/Api.fs [hunk 1]
```diff
@@ -29,14 +29,9 @@ let getCrimeReport postcode = async {
...
-    { WeatherType =
-        weather.ConsolidatedWeather
-        |> Array.countBy(fun w -> w.WeatherStateName)
-        |> Array.maxBy snd
-        |> fst
-        |> WeatherType.Parse
-      AverageTemperature = weather.ConsolidatedWeather |> Array.averageBy(fun r -> float r.TheTemp) }
+let private asWeatherResponse (weather: Weather.OpenMeteoCurrentWeather.CurrentWeather) =
+    { WeatherType = weather.Weathercode |> WeatherType.FromCode
+      Temperature = float weather.Temperature }
```
Comment (sonicbhoc) [inline src/Server/Api.fs:34]: `emptyWeather` still uses the `AverageTemperature` label that was renamed to `Temperature`, which causes the project to fail to build. (Api.fs, line 44)

**Review by sonicbhoc — CHANGES_REQUESTED** [review]
Does not build. `emptyWeather` type must be updated to use `Temperature` instead of `AverageTemperature`.

File: src/Server/Api.fs (hunk 1 above)
Comment (theprash) [reply src/Server/Api.fs:34]: I've fixed this 👍🏾

**Comment by theprash** [comment]
@nickgravgaard Thanks very much for making this change. 🙏🏾

I'll see if I can get it ported into the solution branch today. However, since master isn't even building right now, I'll merge this now, even though the solution will be completely out of sync.

Generally, I think it's easier to make these changes in the solution branch first (because it's easier to test) and them port them to master later so you might want to try that for any future contributions.

**Pull request merged**